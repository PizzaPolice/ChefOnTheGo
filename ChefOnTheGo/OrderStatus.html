<!DOCTYPE html>
<html>
<head>
    <title>Time Remaining</title>

    <!-- Top Navigation -->
    <html>
    <head>
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <style>
            body {margin:0;}

            .topnav {
              overflow: hidden;
              background-color: #333;
            }

            .topnav a {
              float: left;
              display: block;
              color: #f2f2f2;
              text-align: center;
              padding: 14px 16px;
              text-decoration: none;
              font-size: 17px;
            }

            .topnav a:hover {
              background-color: #ddd;
              color: black;
            }

            .active {
              background-color: ##21618C;
              color: white;
            }

            .topnav .icon {
              display: none;
            }

            @media screen and (max-width: 600px) {
              .topnav a:not(:first-child) {display: none;}
              .topnav a.icon {
                float: right;
                display: block;
              }
            }

            @media screen and (max-width: 600px) {
              .topnav.responsive {position: relative;}
              .topnav.responsive .icon {
                position: absolute;
                right: 0;
                top: 0;
              }
              .topnav.responsive a {
                float: none;
                display: block;
                text-align: left;
              }

            }
        </style>
    </head>

    <body>
        <div class="topnav" id="myTopnav">
            <a href="#home" class="active">Home</a>
            <a href="FoodOrder.html">Food Order</a>
            <a href="Checkout.html">Checkout</a>
            <a href="OrderStatus.html">Order Status</a>
            <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="myFunction()">&#9776;</a>
        </div>
        <div style="padding-left:16px">
        </div>
        <script>
        function myFunction() {
            var x = document.getElementById("myTopnav");
            if (x.className === "topnav") {
                x.className += " responsive";
            } else {
                x.className = "topnav";
            }
        }
        </script>
    </body>
</html>


   

    <!--CSS Style Code-->
    <style type="text/css">

                 body{
                     background: #f6f6f6;


                    }


        .title {
            -- height: 60px;
            -- border-bottom: 1px solid #E1E8EE;
            padding: 10px 30px;
            -- color: #5E6977;
            -- font-size: 18px;
            -- font-weight: 400
        }



                     .countdownContainer{
                         position: absolute;;
                         top: 50%;
                         left: 50%;
                         transform : translateX(-50%) translateY(-50%);
                         text-align: center;
                         background: #ddd;
                         border: 1px solid #999;
                         padding: 10px;
                         box-shadow: 0 0 5px 3px #ccc;
                     }


                     .info{
                     font-size: 50px;
                     }

    </style>

</head>



<body>
    <h2 class="title">Time Remaining</h2>
    <table class="countdownContainer">
        <tr class="info">
            <td colspan="4"> Your food will arrive in:</td>
        </tr>
        <tr class="info">
            <td id="days">2</td>
            <td id="hours">3</td>
            <td id="minutes">23</td>
            <td id="seconds">
                2
            </td>
        </tr>
        <tr>
            <td> Days</td>
            <td> Hours</td>
            <td> Minutes</td>
            <td> Seconds</td>
        </tr>
    </table>
    <script type="text/javascript">

        function countdown(){
        var now = new Date();
        var eventDate = new  Date(2018, 0, 31);

        var currentTiime = now.getTime();
        var eventTime = eventDate.getTime();
        var remTime = eventTime - currentTiime;

        var s = Math.floor(remTime / 1000);
        var m = Math.floor(s / 60);
        var h = Math.floor(m / 60);
        var d = Math.floor(h / 24);

        h %= 24;
        m %= 60;
        s %= 60;


        h = (h<10) ? "0" + h : h;
        m = (m<10) ? "0" + m : m;
        s = (s<10) ? "0" + s : s;

        document.getElementById("days").textContent = d;
        document.getElementById("days").innerText = d;

        document.getElementById("hours").textContent = h;
        document.getElementById("minutes").textContent = m;
        document.getElementById("seconds").textContent = s;

        setTimeout(countdown, 1000);

        }

        countdown();

    </script>

</body>
</html>
