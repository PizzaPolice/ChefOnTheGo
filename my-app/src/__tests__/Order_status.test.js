import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render } from 'enzyme';
import OrderStatus from '../components/order_status.jsx'


describe('Login Component', () => {
    it('Login should render without throwing an error', () => {
        expect(shallow(<OrderStatus />).exists(<form className='Login'></form>)).toBe(true)
      })
      it("should render initial layout", () => {
        const component = shallow(<OrderStatus />);
        expect(component.getElements()).toMatchSnapshot();
        })
    });

    describe('Submit Button', () => {
        it('should be defined', () => {
          expect(OrderStatus).toBeDefined();
        });

        it('should render correctly', () => {
          const button = shallow(
            <OrderStatus name='btn next-btn' />
          );
          expect(button).toMatchSnapshot();
        });
    });
