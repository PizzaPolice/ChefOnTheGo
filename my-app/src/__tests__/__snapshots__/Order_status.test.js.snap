// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render initial layout 1`] = `
Array [
  <div
    className="outer-panel"
>
    <div
        className="panel-body"
    >
        <h2
            className="title"
        >
            Your Order Has Been Placed!
        </h2>
        <table
            className="countdownContainer"
        >
            <tr
                className="info"
            >
                <td
                    colspan="3"
                >
                     Your food will arrive in:
                </td>
            </tr>
            <tr
                className="info"
            >
                <td
                    id="hours"
                >
                    20
                </td>
                <td
                    id="minutes"
                >
                    23
                </td>
                <td
                    id="seconds"
                >
                    2
                </td>
            </tr>
            <tr>
                <td>
                     Hours
                </td>
                <td>
                     Minutes
                </td>
                <td>
                     Seconds
                </td>
            </tr>
        </table>
    </div>
    <div
        className="panel-footer"
    >
        <Link
            replace={false}
            to="review"
        >
            <button
                className="btn next-btn"
            >
                Received Order
            </button>
        </Link>
    </div>
</div>,
]
`;

exports[`Submit Button should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OrderStatus
    name="btn next-btn"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="panel-body"
>
          <h2
                    className="title"
          >
                    Your Order Has Been Placed!
          </h2>
          <table
                    className="countdownContainer"
          >
                    <tr
                              className="info"
                    >
                              <td
                                        colspan="3"
                              >
                                         Your food will arrive in:
                              </td>
                    </tr>
                    <tr
                              className="info"
                    >
                              <td
                                        id="hours"
                              >
                                        20
                              </td>
                              <td
                                        id="minutes"
                              >
                                        23
                              </td>
                              <td
                                        id="seconds"
                              >
                                        2
                              </td>
                    </tr>
                    <tr>
                              <td>
                                         Hours
                              </td>
                              <td>
                                         Minutes
                              </td>
                              <td>
                                         Seconds
                              </td>
                    </tr>
          </table>
</div>,
        <div
          className="panel-footer"
>
          <Link
                    replace={false}
                    to="review"
          >
                    <button
                              className="btn next-btn"
                    >
                              Received Order
                    </button>
          </Link>
</div>,
      ],
      "className": "outer-panel",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2
              className="title"
>
              Your Order Has Been Placed!
</h2>,
            <table
              className="countdownContainer"
>
              <tr
                            className="info"
              >
                            <td
                                          colspan="3"
                            >
                                           Your food will arrive in:
                            </td>
              </tr>
              <tr
                            className="info"
              >
                            <td
                                          id="hours"
                            >
                                          20
                            </td>
                            <td
                                          id="minutes"
                            >
                                          23
                            </td>
                            <td
                                          id="seconds"
                            >
                                          2
                            </td>
              </tr>
              <tr>
                            <td>
                                           Hours
                            </td>
                            <td>
                                           Minutes
                            </td>
                            <td>
                                           Seconds
                            </td>
              </tr>
</table>,
          ],
          "className": "panel-body",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Your Order Has Been Placed!",
              "className": "title",
            },
            "ref": null,
            "rendered": "Your Order Has Been Placed!",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <tr
                  className="info"
>
                  <td
                                    colspan="3"
                  >
                                     Your food will arrive in:
                  </td>
</tr>,
                <tr
                  className="info"
>
                  <td
                                    id="hours"
                  >
                                    20
                  </td>
                  <td
                                    id="minutes"
                  >
                                    23
                  </td>
                  <td
                                    id="seconds"
                  >
                                    2
                  </td>
</tr>,
                <tr>
                  <td>
                                     Hours
                  </td>
                  <td>
                                     Minutes
                  </td>
                  <td>
                                     Seconds
                  </td>
</tr>,
              ],
              "className": "countdownContainer",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <td
                    colspan="3"
>
                     Your food will arrive in:
</td>,
                  "className": "info",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " Your food will arrive in:",
                    "colspan": "3",
                  },
                  "ref": null,
                  "rendered": " Your food will arrive in:",
                  "type": "td",
                },
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td
                      id="hours"
>
                      20
</td>,
                    <td
                      id="minutes"
>
                      23
</td>,
                    <td
                      id="seconds"
>
                      2
</td>,
                  ],
                  "className": "info",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "20",
                      "id": "hours",
                    },
                    "ref": null,
                    "rendered": "20",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "23",
                      "id": "minutes",
                    },
                    "ref": null,
                    "rendered": "23",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "2",
                      "id": "seconds",
                    },
                    "ref": null,
                    "rendered": "2",
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <td>
                       Hours
</td>,
                    <td>
                       Minutes
</td>,
                    <td>
                       Seconds
</td>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": " Hours",
                    },
                    "ref": null,
                    "rendered": " Hours",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": " Minutes",
                    },
                    "ref": null,
                    "rendered": " Minutes",
                    "type": "td",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": " Seconds",
                    },
                    "ref": null,
                    "rendered": " Seconds",
                    "type": "td",
                  },
                ],
                "type": "tr",
              },
            ],
            "type": "table",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            replace={false}
            to="review"
>
            <button
                        className="btn next-btn"
            >
                        Received Order
            </button>
</Link>,
          "className": "panel-footer",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <button
              className="btn next-btn"
>
              Received Order
</button>,
            "replace": false,
            "to": "review",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Received Order",
              "className": "btn next-btn",
            },
            "ref": null,
            "rendered": "Received Order",
            "type": "button",
          },
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="panel-body"
>
            <h2
                        className="title"
            >
                        Your Order Has Been Placed!
            </h2>
            <table
                        className="countdownContainer"
            >
                        <tr
                                    className="info"
                        >
                                    <td
                                                colspan="3"
                                    >
                                                 Your food will arrive in:
                                    </td>
                        </tr>
                        <tr
                                    className="info"
                        >
                                    <td
                                                id="hours"
                                    >
                                                20
                                    </td>
                                    <td
                                                id="minutes"
                                    >
                                                23
                                    </td>
                                    <td
                                                id="seconds"
                                    >
                                                2
                                    </td>
                        </tr>
                        <tr>
                                    <td>
                                                 Hours
                                    </td>
                                    <td>
                                                 Minutes
                                    </td>
                                    <td>
                                                 Seconds
                                    </td>
                        </tr>
            </table>
</div>,
          <div
            className="panel-footer"
>
            <Link
                        replace={false}
                        to="review"
            >
                        <button
                                    className="btn next-btn"
                        >
                                    Received Order
                        </button>
            </Link>
</div>,
        ],
        "className": "outer-panel",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="title"
>
                Your Order Has Been Placed!
</h2>,
              <table
                className="countdownContainer"
>
                <tr
                                className="info"
                >
                                <td
                                                colspan="3"
                                >
                                                 Your food will arrive in:
                                </td>
                </tr>
                <tr
                                className="info"
                >
                                <td
                                                id="hours"
                                >
                                                20
                                </td>
                                <td
                                                id="minutes"
                                >
                                                23
                                </td>
                                <td
                                                id="seconds"
                                >
                                                2
                                </td>
                </tr>
                <tr>
                                <td>
                                                 Hours
                                </td>
                                <td>
                                                 Minutes
                                </td>
                                <td>
                                                 Seconds
                                </td>
                </tr>
</table>,
            ],
            "className": "panel-body",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Your Order Has Been Placed!",
                "className": "title",
              },
              "ref": null,
              "rendered": "Your Order Has Been Placed!",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <tr
                    className="info"
>
                    <td
                                        colspan="3"
                    >
                                         Your food will arrive in:
                    </td>
</tr>,
                  <tr
                    className="info"
>
                    <td
                                        id="hours"
                    >
                                        20
                    </td>
                    <td
                                        id="minutes"
                    >
                                        23
                    </td>
                    <td
                                        id="seconds"
                    >
                                        2
                    </td>
</tr>,
                  <tr>
                    <td>
                                         Hours
                    </td>
                    <td>
                                         Minutes
                    </td>
                    <td>
                                         Seconds
                    </td>
</tr>,
                ],
                "className": "countdownContainer",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <td
                      colspan="3"
>
                       Your food will arrive in:
</td>,
                    "className": "info",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": " Your food will arrive in:",
                      "colspan": "3",
                    },
                    "ref": null,
                    "rendered": " Your food will arrive in:",
                    "type": "td",
                  },
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td
                        id="hours"
>
                        20
</td>,
                      <td
                        id="minutes"
>
                        23
</td>,
                      <td
                        id="seconds"
>
                        2
</td>,
                    ],
                    "className": "info",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "20",
                        "id": "hours",
                      },
                      "ref": null,
                      "rendered": "20",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "23",
                        "id": "minutes",
                      },
                      "ref": null,
                      "rendered": "23",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "2",
                        "id": "seconds",
                      },
                      "ref": null,
                      "rendered": "2",
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <td>
                         Hours
</td>,
                      <td>
                         Minutes
</td>,
                      <td>
                         Seconds
</td>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": " Hours",
                      },
                      "ref": null,
                      "rendered": " Hours",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": " Minutes",
                      },
                      "ref": null,
                      "rendered": " Minutes",
                      "type": "td",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": " Seconds",
                      },
                      "ref": null,
                      "rendered": " Seconds",
                      "type": "td",
                    },
                  ],
                  "type": "tr",
                },
              ],
              "type": "table",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              replace={false}
              to="review"
>
              <button
                            className="btn next-btn"
              >
                            Received Order
              </button>
</Link>,
            "className": "panel-footer",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <button
                className="btn next-btn"
>
                Received Order
</button>,
              "replace": false,
              "to": "review",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Received Order",
                "className": "btn next-btn",
              },
              "ref": null,
              "rendered": "Received Order",
              "type": "button",
            },
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
